pygenometracks/tests/test_non_classical_bed.py

```diff
@@ -178,7 +180,7 @@ def test_plot_tracks_bed_unusual_format():
            f"--outFileName {outfile.name}".split()
     pygenometracks.plotTracks.main(args)
     res = compare_images(expected_file,
-                         outfile.name, tolerance)
+                         outfile.name, 19)
```

Comment:
I think it is better use 'tolerance' and either add a specific number, increase tolerance globally  or provide a test file for python 3 (???). 

I thought we only support python 3, therefore I don't understand the comment to this test case (line 169, 170)

Reply:
There is a small change in the font width between python 3.6, python 3.7 and python 3.8. Good idea to add a number to tolerance.